dice.js

```diff
@@ -0,0 +1,6 @@
+const rolldice = function() {
+    let die1 = Math.floor(Math.random() * (6 - 1 + 1)) + 1;
```

Comment:
you can also  do the random function like this 
`Math.floor(Math.random()*6)+1;`